fileHeader.go

```diff
@@ -106,7 +106,7 @@ func (fh *FileHeader) setRecordType() {
 		return
 	}
 	fh.recordType = "01"
-	fh.StandardLevel = "35"
+	// fh.StandardLevel = "35"
```

Comment:
Perhaps this should be moved into `NewFileHeader()` (line 101). 

I'm curious, did you see this value being overridden? When a file is read in this value gets replaced by whatever's in the file (in the `Parse()` function. That said, it does seem odd to me that we're currently setting StandardLevel inside `setRecordType()`.

Reply:
Is this a different value in some specification versions? It's exported to be changed if needed in a specific application.

Reply:
```
‘03’ By clearing arrangements
‘30’ X9.100-187-2008
‘35’ X9.100-187-2013 and 2016
```

Reply:
Yes, I did see this getting overwritten when using the JSON API to create an X9. We are stuck using standard level 03, and that comes back from Moov in both the JSON response and the generated X9 file as 35, this change resolved that.

I tried moving it to line 101 and that did preserve the value sent through the API while retaining 35 as a default. I can make that switch.

Reply:
Oh ok. Thanks for the info.

I bet that means `setRecordType()` is being called somewhere we didn't anticipate. I started looking through those records while reviewing this PR and it seems like we're setting additional values beyond recordType in each `setRecordType()` func.

I'm thinking we should go through each one and make sure they only set `recordType`. I'll create an issue for that.